lib/v2/cmpxchg8b/articles.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const baseUrl = 'https://lock.cmpxchg8b.com/';
+const title = 'cmpxchg8b';
+
+module.exports = async (ctx) => {
+    const { data: response } = await got({
+        method: 'get',
+        url: baseUrl,
+    });
+
+    const $ = cheerio.load(response);
+    const author = $('p.author').text().trim();
+    const list = $('section#articles section')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.find('li').text(),
+                link: new URL(item.find('li a').attr('href'), baseUrl).toString(),
+                author,
+            };
+        });
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+
+                // after #TOC but before footer
+                item.description = $('nav#TOC').nextUntil('footer').html();
```

Comment:
This does not work in the latest 3 articles as you can see from the [preview](https://github.com/DIYgod/RSSHub/pull/13802#issuecomment-1812528568).

Reply:
Thanks.

Fixed by removing unwanted elements from the body explicitly.

```js
  // extract the body but removing <nav> and <footer>
  const body = $('body');
  body.find('nav, footer').remove();
  item.description = body.html();
  return item;
```